types/context_test.go

```diff
@@ -57,6 +57,82 @@ func (s *contextTestSuite) TestCacheContext() {
 	s.Require().Len(ctx.EventManager().Events(), 2)
 }
 
+func (s *contextTestSuite) TestCacheContextIndexerEvents() {
+	key := types.NewKVStoreKey(s.T().Name() + "_TestCacheContextIndexerEvents")
+	k1 := []byte("hello")
+	v1 := []byte("world")
+	k2 := []byte("key")
+	v2 := []byte("value")
+	blockTime := time.Unix(1650000000, 0).UTC()
+	blockHeight := int64(5)
+
+	ctx := testutil.DefaultContext(key, types.NewTransientStoreKey("transient_"+s.T().Name())).
+		WithBlockHeight(blockHeight).
+		WithBlockTime(blockTime).
+		WithIndexerBlockEventManager(types.NewIndexerBlockEventManager(
+			uint32(blockHeight), blockTime),
+		)
+	store := ctx.KVStore(key)
+	store.Set(k1, v1)
+	s.Require().Equal(v1, store.Get(k1))
+	s.Require().Nil(store.Get(k2))
+
+	cctx, write := ctx.CacheContext()
+	cstore := cctx.KVStore(key)
+	s.Require().Equal(v1, cstore.Get(k1))
+	s.Require().Nil(cstore.Get(k2))
+
+	// add some txn events
+	txHash := "txHash"
```

Comment:
nit: Possible to add constants to `testutil/constants` so they can be re-used?

Reply:
done